modules/http/src/main/java/org/mule/module/http/internal/request/RequestResourcesUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.http.internal.request;
+
+import org.mule.module.http.internal.domain.InputStreamHttpEntity;
+import org.mule.module.http.internal.domain.request.HttpRequest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utils to manage request resources
+ * 
+ * @since 3.9.10
```

Comment:
Check the version, also, we will likely need to cherry pick this

Reply:
I've updated the date. I'll take this into account to use the correct version on cherry-picking.